docs/guides/hapi-plugin.rst

```diff
@@ -50,7 +50,7 @@ The available options are
 +-------------------------+---------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | **client**              | ``{}``        | `Hoodie Client`_ options. ``client.url `` is set based on hapi’s ``server.info.host``                                                                                                                       |
 +-------------------------+---------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| **account**             | ``{}``        | `Hoodie Account Server`_ options. ``account.admins``, ``account.secret`` and ``account.usersDb`` are set based on ``db`` option above                                                                       |
+| **account**             | ``{}``        | `Hoodie Account Server`_ options. ``hoodie.account.admins``, ``hoodie.account.secret`` and ``hoodie.account.usersDb`` are set based on ``db`` option above                                                                       |
```

Comment:
I’m sorry but that was actually correct as is, e.g.`account.admins` is a shortcut for the `{account: {admins: ...}}` option. 

But there are several code examples below that use `account` instead of `hoodie.account`, e.g. this line

```js
var properties = account.profile.get()
```

should be 

```js
var properties = hoodie.account.profile.get()
```

Does that make sense?